gsoc/2022/mentors.md

```diff
@@ -15,3 +15,6 @@ layout: plain
 * Javier Lopez-Gomez [Javier Lopez-Gomez](mailto:j.lopez@cern.ch) CERN
 * Dalila Salamani [Dalila Salamani](mailto:dalila.salamani@cern.ch) CERN
 * Anna Zaborowska[Anna Zaborowska](mailto:anna.zaborowska@cern.ch) CERN
+* Benjamin Fuks [fuks@lpthe.jussieu.fr](mailto:fuks@lpthe.jussieu.fr) LPTHE
```

Comment:
sorry, did not spot this earlier. Could you try to put your names in the alphabetical order and insert them appropriately?

Reply:
updated accordingly